src/mixins/linkConfig.js

```diff
@@ -235,6 +220,16 @@ export default {
         this.listeningToMouseup = false;
       }
     },
+    isPoolOrLane(model) {
+      if (!model.component) {
+        return false;
+      }
+      return ['processmaker-modeler-lane', 'processmaker-modeler-pool'].includes(model.component.node.type);
+    },
+    resetBodyColor(shape) {
+      const defaultColor = this.isPoolOrLane(shape) ? poolColor : defaultNodeColor;
+      this.setBodyColor(shape.attr('body/originalFill') || defaultColor, shape);
```

Comment:
This seems to be a direct copy and paste from boundaryEvent.vue. Can we avoid that?